python/lsst/pipe/tasks/mocks/mockCoadd.py

```diff
@@ -197,8 +197,8 @@ def buildInputImages(self, butler, obsCatalog=None, truthCatalog=None, tract=0):
                 if status:
                     simSrcRecord = simSrcCatalog.addNew()
                     simSrcRecord.setCoord(truthRecord.getCoord())
-                    simSrcRecord.setL(self.objectIdKey, truthRecord.getId())
-                    simSrcRecord.setL(self.exposureIdKey, obsRecord.getId())
+                    simSrcRecord.setI(self.objectIdKey, truthRecord.getId())
+                    simSrcRecord.setI(self.exposureIdKey, obsRecord.getId())
```

Comment:
Have you verified that integers can hold the values we use, and won't be too short?